jumanji/wrappers.py

```diff
@@ -618,21 +614,21 @@ def reset(key: chex.PRNGKey) -> Tuple[State, Observation, Optional[Dict]]:
 
         def step(
             state: State, action: chex.Array
-        ) -> Tuple[State, Observation, chex.Array, bool, Optional[Any]]:
+        ) -> Tuple[State, Observation, chex.Array, chex.Array, chex.Array, Optional[Any]]:
             """Step function of a Jumanji environment to be jitted."""
             state, timestep = self._env.step(state, action)
-            done = jnp.bool_(timestep.last())
-            return state, timestep.observation, timestep.reward, done, timestep.extras
+            term = timestep.discount.astype(bool)
+            trunc = timestep.last().astype(bool)
+            return state, timestep.observation, timestep.reward, term, trunc, timestep.extras
 
         self._step = jax.jit(step, backend=self.backend)
 
     def reset(
         self,
         *,
         seed: Optional[int] = None,
-        return_info: bool = False,
         options: Optional[dict] = None,
```

Comment:
Options is to stay in line with the gymnasium API none of our envs accept options, so we can't use it. But if we remove it might not work with other gymnasium wrappers

Reply:
Ah I see now thanks!